src/partials/sidebar/components/category.js

```diff
@@ -1,47 +1,37 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { isEmpty } from "lodash";
+import isEmpty from "lodash/isEmpty";
 
 import {
   SidebarSectionHeading,
   SidebarSectionList,
   SidebarSectionSublist
 } from "../styles";
 
-class Category extends React.Component {
-  renderSubCategories(subCategories) {
-    if (isEmpty(subCategories)) {
-      return null;
-    }
-    const categories = subCategories.map((category, index) =>
-      !isEmpty(category.content) ? (
-        <div key={index}>
-          <SidebarSectionHeading>{category.title}</SidebarSectionHeading>
-          <SidebarSectionSublist>{category.content}</SidebarSectionSublist>
-        </div>
-      ) : null
-    );
-    return categories.filter(Boolean);
+const Category = ({ content, title, subCategories }) => {
+  if (isEmpty(subCategories) || isEmpty(content)) {
```

Comment:
I think this might be the cause of the searching issue:

Because [here](https://github.com/FormidableLabs/victory-docs/blob/af83655be6525f22d8d99188bc0253c48f8cbb5d/src/partials/sidebar/index.js#L225) we only ever pass the `linksList.documentation` this is triggering `isEmpty(content)` to return true

Reply:
oh my gosh thank you for catching this!